docs/getting-started/proofs-and-verification/gosdk/state.md

```diff
@@ -2,4 +2,30 @@
 id: state
 title: Proof of Existence of a State/Data
 ---
+### State proof example
+Attached simple code snippet to show how to use ledger API for state existences proof:
```

Comment:
Below is a simple code snippet that shows how to use the ledger API in order to generate a state existence proof: